testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/ProfileAssume.java

```diff
@@ -23,32 +23,30 @@
 import org.keycloak.representations.info.ProfileInfoRepresentation;
 import org.keycloak.testsuite.util.AdminClientUtil;
 
-import java.util.Collections;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Set;
 
 /**
  * @author <a href="mailto:sthorger@redhat.com">Stian Thorgersen</a>
  */
 public class ProfileAssume {
 
-    private static Set<String> disabledFeatures;
+    private static List<String> disabledFeatures;
     private static String profile;
 
     private static void updateProfile() {
```

Comment:
This used to load the profile from the server every time isFeatureEnable was called, which was initializing the admin client, fetching tokens, and loading server info many times during tests. Kinda pointless so changed this to load once, then it will be updated as features are enabled/disabled through the testing client.